objdb/client.go

```diff
@@ -24,6 +24,21 @@ import (
 
 var defaultDbURL = "etcd://127.0.0.1:2379"
 
+// InitClient aims to support multi endpoints
+func InitClient(storeName string, storeURLs []string) (API, error) {
+	plugin := GetPlugin(storeName)
+	if plugin == nil {
+		log.Errorf("Invalid DB type %s", storeName)
```

Comment:
I'd use`datastore` instead of `DB` :)

Reply:
The `NewClient` is using the DB name, here is just make them consistent. meanwhile the package named `objdb` so i think db is not a terrible name